posthog/api/capture.py

```diff
@@ -526,6 +531,7 @@ def is_randomly_partitioned(candidate_partition_key: str) -> bool:
                 # Return early if we have logged this key already.
                 return True
 
+            PARTITION_KEY_CAPACITY_EXCEEDED_COUNTER.labels(partition_key=candidate_partition_key).inc()
             statsd.incr("partition_key_capacity_exceeded", tags={"partition_key": candidate_partition_key})
```

Comment:
Should we get rid of the statsd call now that we have the prom metric?

Reply:
Might as well, yes!

Reply:
I'll do a PR to mirror the other metrics to prom, I'll delete that metric in that PR instead of this one.